### PR TITLE
Create and integrate PoemBot poem translations

### PR DESCRIPTION
**Pull request by smusoke**
**What**:
- Converted poem constants into translations
- Updated the `PoemSelector` component to use keys as a unique identifier, instead of poem title (this is translated)

**Why**: We plan to translate the poems moving forward. The next i18n sync should push these new translations to crowdin.

## Links

- jira ticket: [FND-1755](https://codedotorg.atlassian.net/browse/FND-1755)

## Testing story

Tested locally by opening the same lesson in production. All functionality was equivalent on my end.

Screen Shot 2021-10-14 at 12 50 00 PM

Screen Shot 2021-10-14 at 12 50 30 PM

## Follow-up work

Create documentation on how to add new content to the i18n sync.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/p5lab/poetry/poem.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import msg from '@cdo/poetry/locale';
+import {POEMS} from './constants';
+
+export function getPoem(key) {
```
Comment (smusoke) [inline apps/src/p5lab/poetry/poem.js:4]: Didn't see a pattern to follow in terms of grouping logic that's shared, so created this individual poem file. Let me know if there is a different convention we try to follow.

File: apps/i18n/poetry/en_us.json [hunk 2]
```diff
@@ -1,40 +1,82 @@
...
-  "successLevel6": "Congratulations! You completed Level 6.",
-  "successLevel7": "Congratulations! You completed Level 7.",
-  "successLevel8": "Congratulations! You completed Level 8.",
-  "successLevel9": "Congratulations! You completed Level 9.",
-  "successLevel10": "Congratulations! You completed Level 10.",
-  "successRemovedSprite": "Great work removing a new sprite with an event! Click “Keep Playing” to try adding  a sprite with an event.",
-  "successTextEvent": "Great job changing your text effect with an event! Click 'Keep Playing' to try changing the foreground or background effect.",
-  "textEffectWithEvent": "Set a new text effect under your `when line shows` event block.",
-  "title": "Title"
+	"alexanderLines": "What happened to a dream destroyed?\n\nDoes it sink\nlike a wrecked ship in the sea?\n\nOr wade in the water\nLike a boy overboard?\n\nMaybe it just floats\nAround and around...\n\nOr does it drown?",
```
Comment (ajpal) [inline apps/i18n/poetry/en_us.json]: nit: looks like the indentation has all changed?

**Review by ajpal — APPROVED** [review]
thank you for doing this!! one comment about spacing, but otherwise lgtm!

File: apps/i18n/poetry/en_us.json (hunk 2 above)
Comment (smusoke) [reply apps/i18n/poetry/en_us.json]: Good eye!! Got this fixed 👍

**Pull request merged**